src/common/PopoverItem.js

```diff
@@ -22,10 +29,34 @@ class PopoverItem extends Component {
         };
     }
 
+    /**
+     * Popover toggle
+     *
+     * @param  {popOverOpen}
+     * @param  isActive {bool} - from parent component Attend received as props
+     * @param  onClick {func} - from parent component received as props
+     */
     toggle() {
-        this.setState({
-            popoverOpen: !this.state.popoverOpen,
-        });
+        /**
+         * Call OnClick function from parent component Attend to change heart icon,
```

Comment:
No need to explicitly name which is the parent component, since in theory can be everything, but it's way better than not documenting it at all 👍